src/test/java/posters/cucumber/features/BrowseRandomVisualAssert.feature

```diff
@@ -1,4 +1,4 @@
-@BrowseRandomVisualAssert
+@Wip @BrowseRandomVisualAssert
```

Comment:
remove `@Wip`. The `@Skip` should be sufficient to Ignore this test.

Reply:
done

Reply:
This annotation is the only one in all .feature files. When we remove it, how should SingleFeatureTest with
tags = "@Wip and not @Skip" work?
When we would use: tags = "not @Skip", all tests would be executed.

Reply:
new feature file added